### PR TITLE
Edit: SFTP and Rcon

### PR DESCRIPTION
**Pull request by Mocab**
Metadata was not correct (i think), the category was different from the link.

### CODE REVIEW

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md [hunk 1]
```diff
@@ -27,12 +27,37 @@ SFTP Clients:
 |:---------|:---------------------|:-------------------------|
 | [Filezilla](https://filezilla-project.org/download.php?type=client) | Windows, macOS, and Linux |
 | [Cyberduck](https://cyberduck.io/download/) | Windows and macOS |
-| [WinSCP](https://winscp.net/eng/download.php) | Windows |
+| [WinSCP](https://winscp.net/eng/download.php) (Reccomended)| Windows |
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:30]: 1. Spelling.

```suggestion
| [WinSCP](https://winscp.net/eng/download.php) (Recommended)| Windows |
```

2. Why is WinSCP recommended?

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md [hunk 2]
```diff
@@ -27,12 +27,37 @@ SFTP Clients:
...
 | [Swish](https://sourceforge.net/projects/swish/) | Windows |
+| [AndFTP](https://play.google.com/store/apps/details?id=lysesoft.andftp&hl=en_US&gl=US) | Android |
+| [FTPManager](https://apps.apple.com/us/app/ftpmanager-ftp-sftp-client/id525959186) | IOS |
 
-NOTE: Linux usually has SFTP integration built into the file manager.
+`NOTE: Linux usually has SFTP integration built into the file manager.`
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:35]: Why is this set to inline code?

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md (hunk 1 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:30]: Sometimes filezilla doesn't connect for some reason, while win SCP works fine

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md (hunk 2 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:35]: looks better, should i remove it?

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md (hunk 1 above)
Comment (DaddyChimken) [reply _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:30]: WinSCP is only for Windows. Imo we shouldn't recommended something that only supports one specific OS.

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md (hunk 2 above)
Comment (DaddyChimken) [reply _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:35]: It's not even code, so yes please.

I'd be fine with bolding it though.

File: _posts/minecraft/general/2021-06-23-sftp-and-rcon.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
 title:  "Using SFTP"
-categories: Minecraft
-tag: mcgeneral
+categories: Falix
+tag: fxgeneral
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-06-23-sftp-and-rcon.md:5]: I don't know much on how the help-center works, so this is for @KorbsStudio; Is this tag correct?

**Pull request merged**